kse/src/main/java/org/kse/gui/preferences/data/KsePreferences.java

```diff
@@ -39,7 +40,8 @@ public class KsePreferences {
     private CaCertsSettings caCertsSettings = new CaCertsSettings();
     private KeyGenerationSettings keyGenerationDefaults = new KeyGenerationSettings();
     private DigestType certificateFingerprintAlgorithm = DigestType.SHA1;
-    private PasswordQualityConfig passwordQualityConfig = new PasswordQualityConfig(false, false, 60);
+    private PublicKeyFingerprintAlgorithm publicKeyFingerprintAlgorithm = PublicKeyFingerprintAlgorithm.SKI_METHOD1;
+	private PasswordQualityConfig passwordQualityConfig = new PasswordQualityConfig(false, false, 60);
```

Comment:
The indentation here seems to be too big

Reply:
fixed